### DISCUSSION TITLE
Token regeneration consistently fails

### DISCUSSION THREAD

**Discussion by aendie** [community/community#38543] [Other Feature Feedback, Questions, & Ideas]
I find this very frustrating. In not-so ancient times it did work and I was able to regenerate tokens before expiry. Now all I get in Firefox 106.0.5, Chrome and Edge is this when I click on the regeneration link (after authentication via "Authenticator" on my Samsung Galaxy S20 FE): 
![GitHub fail](https://user-images.githubusercontent.com/13238632/200527423-9a0e1b81-be84-4a50-bba7-086f756af669.JPG)
I have disabled the only extension I use: Intenet Download Manager (IDM). It still fails.
Maybe something is peculiar to my hardware/software setup? I have:

- a Gigabyte X570 Aorus Ultra motherboard with latest F36d BIOS
- AMD Ryzen 7 3700X 8-core processor
- virtualization enabled
- dual boot setup (Windows 10 Pro 21H2 + Ubuntu Desktop 22.04 LTS)

Just ask if you need more information. I use GitHub fairly regularly (on a long timescale).

**UPDATE:**
I regenerated my "classic" access token within GitHub-> Settings-> Developer Settings-> Personal Access Tokens.
So my question above is now academic ... I still think it needs looking into.
**P.S.**
Am I correct in thinking that a Personal Access token is required so that "git push origin master" works?

**Comment by ossanna16** [comment]
Hi there @aendie and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
